internal/apis/kfd/v1alpha2/eks/creator.go

```diff
@@ -24,7 +24,10 @@ import (
 	kubex "github.com/sighupio/furyctl/internal/x/kube"
 )
 
-var ErrUnsupportedPhase = errors.New("unsupported phase")
+var (
+	ErrUnsupportedPhase = errors.New("unsupported phase")
+	ErrInfraNotPresent  = errors.New("cluster configuration does not contain infrastructure section")
```

Comment:
```suggestion
	ErrInfraNotPresent  = errors.New("configuration file does not contain infrastructure section")
```

if we could add also the path to the file, that would be even better

Reply:
now is something like:
```
ERRO error while creating cluster: the configuration file does not contain an infrastructure section: check at /path/to/furyctl.yaml
```